stdpopsim/catalog/AraTha/genetic_maps.py

```diff
@@ -2,10 +2,18 @@
 
 _species = stdpopsim.get_species("AraTha")
 
+_genetic_map_citation = stdpopsim.Citation(
+    doi="https://doi.org/10.1038/hdy.2011.95",
+    author="Salomé et al.",
+    year=2012,
+    reasons={stdpopsim.CiteReason.GEN_MAP},
+)
+
+tarballname = "AraTha/salome2012_liftover_maps.tar.gz"
```

Comment:
URL failed commit checks with full name. Please advise if alternative solution is preferred.

Reply:
which checks? Like, "line too long checks"? If so, you can just do like
```
tarballname = "https://this_is_really.long.url/so/just/"
                        "spread_the_string_over/more_lines.tar.gz"
```